gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""Integration testing runner for Variant Transforms' Preprocessor pipeline.
```

Comment:
nit: Is the "r" prefix needed? I see that it is in the old run_tests.py as well but I don't think it is needed.

Reply:
In [PEP 257](https://www.python.org/dev/peps/pep-0257/), it is suggested: 
`Use r"""raw triple double quotes""" if you use any backslashes in your docstrings.`

Reply:
I see, thanks.